jobs/multi-arch-pipeline.Jenkinsfile

```diff
@@ -285,13 +285,20 @@ stages:
 - id: ExecOrder 1 Stage
   execution_order: 1
   description: Stage 1 execution base
-  build_artifacts: [qemu]
-  require_artifacts: [ostree]
   prep_commands:
     - cat /cosa/coreos-assembler-git.json
+  require_artifacts: [ostree]
   post_commands:
+    - cosa buildextend-qemu
     - cosa kola run --basic-qemu-scenarios
+    - cosa kola run --parallel 4
+    - cosa buildextend-metal
+    - cosa buildextend-metal4k
+    - cosa buildextend-live
+    - kola testiso -S --output-dir tmp/kola-metal
+    - kola testiso -SP --qemu-native-4k --scenarios iso-install --output-dir tmp/kola-metal4k
```

Comment:
Are the output dirs here copied off the worker so we can archive them? Feels like we're also missing the `archiveArtifacts` step here.

Reply:
agree, yes they are copied back. I've got a few more followups to make (kola aws uploads and testing) and some cycles to run in my dev jenkins. Can I add that step then?

Reply:
actually there is some more I need to do to get them to get copied back - but yeah will do that in a followup